auth-api/src/auth_api/models/org.py

```diff
@@ -193,6 +194,15 @@ def search_org(cls, search: OrgSearch):
         query = cls._search_by_business_identifier(query, search.business_identifier)
         query = cls._search_for_statuses(query, search.statuses)
 
+        if search.suspended_date_from:
```

Comment:
might be able to use filter_conditional_date_range() helper. Looks like it is available in CustomQuery, but we haven't used it in auth yet.

We use this in pay